src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
+    }
```

Comment:
creation of util classes should be forbidden or impossible
```suggestion
    private HibernateUtil() {
    }
```